### PR TITLE
fix CI + cleanup

### LINKED ISSUE #9: Travis is red
ping @agramfort @lucyleeow

### PR DESCRIPTION
**Pull request by agramfort**
fixes #9

### CODE REVIEW

**Comment by agramfort** [comment]
green !

File: submissions/use_external_data/estimator.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (glemaitre) [inline submissions/use_external_data/estimator.py:1]: No need

File: submissions/use_external_data/estimator.py [hunk 2]
```diff
@@ -13,11 +14,12 @@ def _merge_external_data(X):
         os.path.dirname(__file__), 'external_data.csv'
     )
     # Make sure that DateOfDeparture is of dtype datetime
+    X = X.copy()  # modify a copy of X
```
Comment (glemaitre) [inline submissions/use_external_data/estimator.py:17]: Why a copy?

Comment (agramfort) [reply submissions/use_external_data/estimator.py:17]: I get pandas warnnings otherwise as I modify a slice of a dataframe

**Pull request merged**